charts/newrelic-infra-operator/values.yaml

```diff
@@ -14,7 +14,7 @@
 
 image:
   repository: newrelic/newrelic-infra-operator
-  tag: ""  # Defaults to AppVersion
+  tag: "" # Defaults to AppVersion
```

Comment:
Missing space before comment

Reply:
Thanks, corrected them and did a new push